src/main/java/net/javadiscord/javabot/systems/help/AutoCodeFormatter.java

```diff
@@ -118,6 +125,9 @@ void handleMessageEvent(@Nonnull MessageReceivedEvent event, boolean isFirstMess
 	private void sendFormatHint(MessageReceivedEvent event) {
 		event.getMessage()
 				.replyEmbeds(formatHintEmbed(event.getGuild()))
+				.addActionRow(
```

Comment:
Why don't you just use `InteractionUtils.DELETE_ORIGINAL_TEMPLATE`?

Reply:
I am doing that already, no?

Reply:
So what's the point of `FORMAT_HINT_IDENTIFIER` and `handleDeleteHint`?